_includes/js/queries.md

```diff
@@ -537,3 +537,15 @@ query.distinct("age")
     // There was an error.
   });
 ```
+
+## Read Preference
+
+When using a MongoDB replica set, you can use the `query.readPreference(readPreference, includeReadPreference, subqueryReadPreference)` function to choose from which replica the objects will be retrieved. The `includeReadPreference` chooses from which replica the included pointers will be retrieved and the `subqueryReadPreference` chooses in which replica the subqueries will run. The possible values are `PRIMARY` (default), `PRIMARY_PREFERRED`, `SECONDARY`, `SECONDARY_PREFERRED`, or `NEAREST`. If the `includeReadPreference` is not passed, the same replica chosen for `readPreference` will be also used for the includes. The same rule applies for the `subqueryReadPreference`.
```

Comment:
Maybe... “The includesReadPreference chooses” would read better as “The includesReadPreference argument/parameter chooses”

Reply:
Same for the others as well.

Reply:
done